README.md

```diff
@@ -22,6 +22,15 @@ This repository contains a number of additional integrators for OpenMM in `openm
 
 The `openmmtools.testsystems` module contains a large suite of test systems---including many with simple exactly-computable properties---that can be used to test molecular simulation algorithms
 
+## Monte Carlo moves
```

Comment:
Let's call this "Markov chain Monte Carlo proposal schemes and move compositions" or something.